src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -411,13 +411,18 @@ export class CellExecution {
             return this.completedSuccessfully().then(noop, noop);
         }
 
+        // For Jupyter requests, silent === don't output, while store_history === don't update execution count
+        // https://jupyter-client.readthedocs.io/en/stable/api/client.html#jupyter_client.KernelClient.execute
+        // for our usage (such as variable requests) we do want output, but we don't want to update execution count
+        const executeSilently = false;
+
         const request = session.requestExecute(
             {
                 code,
                 silent: false,
```

Comment:
Shouldn't we then use `executeSilently` here.

Reply:
No, not here, because it's the mismatch between our concept of silent and the JMC message concept. But shouldn't matter as I'm going to change this per my above comment.